chart/values.yaml

```diff
@@ -724,3 +728,4 @@ storageClasses:
   - name: hcloud-volumes
     defaultStorageClass: true
     reclaimPolicy: Delete
+    includesRobotServers: false 
```

Comment:
Can you add some documentation to this value to explain what it does and that it is immutable (ie. can only be set when creating a storage class, not modified later on).

Reply:
Added parameter documentation to the helm values and the readme.md.